packages/app/src/domain/topical/topical-page-header.tsx

```diff
@@ -13,12 +12,14 @@ interface TopicalPageHeaderProps {
   title: ReactNode;
   lastGenerated: number;
   showBackLink?: boolean;
+  link?: ReactNode;
```

Comment:
We can make it more clear this is a render prop:
```suggestion
  renderLink?: ReactNode;
```

Reply:
Orrr .. wait, this doesn't make sense because it isn't a render prop at all.

Im only confused by the name of the `prop` being `link` which could be a lot things..

Reply:
Would `linkComponent` be more appropriate?

Reply:
Yes that would be a good name I think, but I'm about to replace it with text/href after this so don't worry 😅 I believe the Link layout is standard for all section headings and therefor the header should instantiate the link component. I am also adding the text and href for each section to lokalize.